torch/csrc/jit/passes/dead_code_elimination.cpp

```diff
@@ -100,7 +100,8 @@ class DeadCodeEliminator {
       // If there's no outer node, we're looking at the graph's top-level
       // return block. We consider all graph outputs to be "used", so just mark
       // this node normally.
-      return mark(node);
+      mark(node);
```

Comment:
I don't quite like the fact that `markReturnNode` isn't changed to `bool`. I get that we only call `mark` for the top-level (function return) and in this case we aren't in a loop, so it doesn't matter, and if we are in a loop we make live only body outputs. When we'll get early returns we might revisiting this and make `markReturnNode` to actually return a `bool`